i18n/ui/lbm-ui_en-US.po

```diff
@@ -647,8 +687,8 @@ msgid "We see your transaction but we're still waiting for confirmation. This ta
 msgstr "We see your transaction but we're still waiting for confirmation. This takes 10 minutes on average but can take up to an hour or two."
 
 #: On screen: unconfirmed-deposit
-msgid "Please come back for your cash and press <strong>Redeem</strong> when you get our text message."
-msgstr "Please come back for your cash and press <strong>Redeem</strong> when you get our text message."
```

Comment:
Why are we removing this tag?

Reply:
The tag doesn't really work with the translation tools.